src/deadline/client/cli/_groups/click_logger.py

```diff
@@ -0,0 +1,39 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+import click
+import json
+import typing as t
+
+
+class ClickLogger:
```

Comment:
I made a dedicated logger class to abstract out plain print vs json print. This allows us devs to print whenever w/o worrying about JSON or test mode.